--- gtkam-0.1.13/configure.in.orig	2006-01-09 13:14:57.000000000 +0100
+++ gtkam-0.1.13/configure.in	2006-01-09 13:15:29.000000000 +0100
@@ -58,7 +58,7 @@
 dnl i18n support
 dnl ---------------------------------------------------------------------------
 GP_GETTEXT_HACK([],[Lutz Müller and others],[${MAIL_GPHOTO_TRANSLATION}])
-ALL_LINGUAS="cs da de es fr hu it ja nb pl pt_BR pt_PT ru sl sv uk"
+ALL_LINGUAS="cs da de es fr hu it ja nb pl pt_BR pt ru sl sv uk"
 AM_GNU_GETTEXT_VERSION([0.14.1])
 AM_GNU_GETTEXT([external])
 AM_PO_SUBDIRS()
