diff -urN aa/gtkam-0.1.11/configure.in gtkam-0.1.11/configure.in
--- aa/gtkam-0.1.11/configure.in	2004-03-06 23:35:26.000000000 +0100
+++ gtkam-0.1.11/configure.in	2004-03-21 13:19:24.917621312 +0100
@@ -16,7 +16,7 @@
 AC_DEFINE_UNQUOTED(HAVE_CC,"$CC",[The C compiler we're using])
 
 dnl Add the languages which your application supports here.
-ALL_LINGUAS="cs da de es fr hu it ja no pt_BR pt_PT ru sl sv uk"
+ALL_LINGUAS="cs da de es fr hu it ja nb pt_BR pt_PT ru sl sv uk"
 AM_GNU_GETTEXT
 
 dnl ---------------------------------------------------------------------------
