--- gtkam-0.1.15/help/C/Makefile.am.orig	2007-08-12 10:30:50.000000000 +0200
+++ gtkam-0.1.15/help/C/Makefile.am	2008-05-29 22:52:54.739387284 +0200
@@ -1,11 +1,4 @@
-figdir =
-figdir += figures/gtkam_add_camera_enhanced_window.png
-figdir += figures/gtkam_add_camera_window.png
-figdir += figures/gtkam_fileinfo.png
-figdir += figures/gtkam_makedir.png
-figdir += figures/gtkam_save_all.png
-figdir += figures/gtkam_save.png
-figdir += figures/gtkam_start_window.png
+figdir = figures
 
 docname = gtkam
 lang = C
--- gtkam-0.1.15/help/ru/Makefile.am.orig	2007-08-12 10:53:08.000000000 +0200
+++ gtkam-0.1.15/help/ru/Makefile.am	2008-05-29 22:53:05.896023065 +0200
@@ -1,11 +1,4 @@
-figdir = 
-figdir += figures/gtkam_add_camera_enhanced_window.png
-figdir += figures/gtkam_add_camera_window.png
-figdir += figures/gtkam_fileinfo.png
-figdir += figures/gtkam_makedir.png
-figdir += figures/gtkam_save_all.png
-figdir += figures/gtkam_save.png
-figdir += figures/gtkam_start_window.png
+figdir = figures
 
 docname = gtkam
 lang = ru
